components/scream/src/share/io/scorpio_input.hpp

```diff
@@ -33,43 +25,34 @@
  *  this class to facilitate cases where reading input over some number of simulation
  *  timesteps is possible.
  *
- *  At construction time All input instances require
+ *  At construction time input instances require
  *  1. an EKAT comm group and
  *  2. a EKAT parameter list
  *  3. a shared pointer to the field manager
- *  4. a shared pointer to the grids manager
  *  The parameter list contains all of the essential data regarding
  *  the input file name and the set of variables to be read.  The parameter list can be
- *  created localling in the process needing input, see src/share/io/tests/ for examples of
+ *  created locally in the process needing input, see src/share/io/tests/ for examples of
  *  setting up the input parameter list.
  *
  *  A typical input parameter list looks like:
  *  -----
  *  Input Parameters
  *    FILENAME: STRING
  *    GRID: STRING
- *    FIELDS
- *      Number of Fields: INT
- *      field_1: STRING
- *      ...
- *      field_N: STRING
+ *    FIELDS: [field_name_1, field_name_2, ..., field_name_N]
```

Comment:
If people would rather have one field name per line (easier to insert/delete fields), one can use hyphens:

```
FIELDS:
    - field_name_1
    - field_name_2
    ...
    - field_name_N
```

Reply:
Does that still map to a sequence, like the syntax I used in the comment? Nice!

Reply:
Yup. It's alternate syntax for a YAML sequence.